test/utils/setup.ts

```diff
@@ -139,16 +135,20 @@ export const getMock = async () => {
 
 export const getSafeTemplate = async (saltNumber: string = getRandomIntAsString()) => {
     const singleton = await getSafeSingleton();
+    return getSafeTemplateWithSingleton(singleton, saltNumber);
+};
+
+export const getSafeTemplateWithSingleton = async (singleton: Contract | Safe, saltNumber: string = getRandomIntAsString()) => {
     const singletonAddress = await singleton.getAddress();
     const factory = await getFactory();
     const template = await factory.createProxyWithNonce.staticCall(singletonAddress, "0x", saltNumber);
     await factory.createProxyWithNonce(singletonAddress, "0x", saltNumber).then((tx: any) => tx.wait());
-    const Safe = await getSafeSingletonContractFactoryFromEnvVariable();
```

Comment:
Oh, looks like it was used here, but the refactor simplifies is more elegant than before.